docs/cloud-providers/gcp/gcp-provider.md

```diff
@@ -14,29 +14,63 @@ You can choose whichever you are more comfortable with.
 
 ## Option 1: gcloud Command Line Tool
 
-If you have the `gcloud` tool installed, you can run below commands from the example directory.
-It
+If you have the `gcloud` tool installed, you can run the commands below from the crossplane directory.
+
 Instructions for installing `gcloud` can be found in the [Google docs](https://cloud.google.com/sdk/install).
 
+### Using `gcp-credentials.sh`
+
+In the `cluster/examples` directory you will find a helper script, `gcp-credentials.sh`.  This script will prompt you for the organization, project, and billing account that will be used by `gcloud` when creating a project, service account, and credentials file (`crossplane-gcp-provider-key.json`).  The chosen project and created service account will have access to the services and roles sufficient to run the Crossplane GCP examples.
+
+```console
+$ cluster/examples/gcp-credentials.sh
+...
+#
+# Run the following for the variables that are used throughout the GCP example projects
+#
+export ORGANIZATION_ID=987654321
```

Comment:
after writing a lot of docs over the past few years, i've noticed that folks have a tendency to copy/paste everything they see.  i'm wondering if folks are blindly going to copy/paste these sample export commands.  Should we not include the sample output here, or maybe make it more obvious that this whole thing is just an example? What's your take here?

Reply:
I'll give you a shiny nickel the first time that happens. :nerd_face: 

I could go either way.  This example reflects something a little more like what the actual output would look like.  The markdown convention for a command with its output is to start with the `$`, I'm hoping that the first line is more appealing than the remainder of the console text.

In any case, I've refined that section a bit in e6cdbb4a25125a4081fa65bf0fd6aea8702ca15d by removing the "Run the following" line of text.

I can use `<YOUR ORGANIZATION ID HERE>` if it seems better.

---

```console
$ export ORGANIZATION_ID=<YOUR ORGANIZATION ID HERE>
bash: YOUR: No such file or directory
```

Reply:
cool, i think it looks good now the way you've tightened it up in the last commit.  Thanks!